src/lib/homestore_backend/hs_pg_manager.cpp

```diff
@@ -174,14 +176,23 @@ HSHomeObject::HS_PG::HS_PG(PGInfo info, shared< homestore::ReplDev > rdev, share
         pg_sb_->members[i].priority = m.priority;
         ++i;
     }
-
     pg_sb_.write();
+    init_cp();
+}
+
+void HSHomeObject::HS_PG::init_cp() {
+    cache_pg_sb_ = (pg_info_superblk*)malloc(pg_sb_->size());
+    cache_pg_sb_->copy(*(pg_sb_.get()));
```

Comment:
i prefer to remove `copy` , and overwrite operator new for `pg_info_superblk`. i think it will be more readable